### DISCUSSION TITLE
Modal.confirm 的okText不是默认的确定两字

### DISCUSSION THREAD

**Discussion by ghost** [vueComponent/ant-design-vue#6651] [General]
![image](https://github.com/vueComponent/ant-design-vue/assets/37528714/3bd4bfdf-1c96-4fdf-9883-d8038b4a1ba1)

![image](https://github.com/vueComponent/ant-design-vue/assets/37528714/9072bfed-b4e0-4d3f-8576-8127c28a8ab3)

**Comment by daiwanxing** [comment]
@LiuChang0725 在以函数的形式调用 `<a-modal>` 时只能先手动声明 `okText` 作为临时解决办法
